docs/configs/frontera.py

```diff
@@ -21,22 +20,21 @@
             max_workers_per_node=2,
             address=address_by_hostname(),
             provider=SlurmProvider(
-                cmd_timeout=60,     # Add extra time for slow scheduler responses
+                cmd_timeout=60,  # Add extra time for slow scheduler responses
                 nodes_per_block=2,
                 init_blocks=1,
                 min_blocks=0,
                 max_blocks=1,
-                partition=user_opts['frontera']['partition'],  # Replace with partition name
-
+                partition=user_opts["frontera"][
+                    "partition"
+                ],  # Replace with partition name
                 # Enter scheduler_options if needed
-                scheduler_options=user_opts['frontera']['scheduler_options'],
-
```

Comment:
Removing these newlines is maybe not the best idea. The lines are there to group comments and options logically for the user.